boards/makerbase/mks_canable_v20/mks_canable_v20.dts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/g4/stm32g431X8.dtsi>
+#include <st/g4/stm32g431c(6-8-b)tx-pinctrl.dtsi>
+
+/ {
+	model = "Makerbase MKS CANable V2.0";
+	compatible = "makerbase,mks-canable-v20";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,canbus = &fdcan1;
+	};
+
+	aliases {
+		led0 = &blue_led;
+		led1 = &green_led;
+		mcuboot-led0 = &blue_led;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		blue_led: led_2 {
+			gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+			label = "blue-status D2";
+		};
+		green_led: led_3 {
+			gpios = <&gpioa 0 GPIO_ACTIVE_LOW>;
+			label = "green-word D3";
+		};
+	};
+
+	aliases {
+		led0 = &green_led;
+	};
+};
+
+&clk_hse {
+	status = "disabled";
+};
```

Comment:
Not required, hse (and clocks in general) is disabled by default

Reply:
done in 89b786829c8f646f7c172890186c67f52a357ce4